tools/mock-service-host/src/webserver/validateAndMockController.ts

```diff
@@ -110,7 +110,14 @@ export class ValidateAndMockController extends BaseHttpController {
             for (const name in response.headers) {
                 res.setHeader(name, response.headers[name])
             }
-            return res.status(parseInt(response.statusCode)).json(response.body)
+            if (req.headers?.accept?.startsWith('text/')) {
+                return res
+                    .status(parseInt(response.statusCode))
+                    .contentType(req.headers?.accept)
+                    .send(response.body)
+            } else {
+                return res.status(parseInt(response.statusCode)).json(response.body)
+            }
```

Comment:
I tend to keep json as the default content-type since: 1) it's used in great majority of operations. 2) it bring least impact comparing with previous ALL-JSON solution.